src/icons/file/IconFileAudio.js

```diff
@@ -1,34 +1,23 @@
 // @flow
 import * as React from 'react';
 
-import AccessibleSVG from '../accessible-svg';
+import IconFileBase from './IconFileBase';
+import type { Icon } from '../flowTypes';
 
-type Props = {
-    className?: string,
-    height?: number,
-    /** A text-only string describing the icon if it's not purely decorative for accessibility */
-    title?: string | React.Element<any>,
-    width?: number,
+const IconFileAudio = (props: Icon) => {
+    return (
```

Comment:
Can we stick with implicit returns? It looks like doing so would eliminate most of the whitespace changes in this changeset.

Reply:
Fixed.